### ISSUE TITLE
UnhandledPromiseRejectionWarning: RepositoryNotFoundError: No repository for "CredentialsEntity" was found.

### ISSUE THREAD

**Issue by wowi42** [n8n-io/n8n#776]
**Describe the bug**
Trying to deploy n8n with PostgreSQL, using the last version.

Silently fail, so I had to modify some code. After modifying ```packages/cli/src/Db.ts```

```
-
-       const connection = await createConnection(connectionOptions);
-
-       await connection.runMigrations({
-               transaction: 'none',
-       });
-
+       console.log(connectionOptions);
+       console.log(connectionOptions.entities);
+       createConnection(connectionOptions).then(async connection => {
+               await connection.runMigrations({
+                       transaction: 'none',
+               });
+       }).catch(error => console.log(error));
```

I got a clear message:

```
(node:31900) UnhandledPromiseRejectionWarning: RepositoryNotFoundError: No repository for "CredentialsEntity" was found. Looks like this entity is not registered in current "default" connection?
    at new RepositoryNotFoundError (/home/loictosser/github.com/n8n/n8n/packages/cli/src/error/RepositoryNotFoundError.ts:10:9)
    at EntityManager.getRepository (/home/loictosser/github.com/n8n/n8n/packages/cli/src/entity-manager/EntityManager.ts:1204:19)
    at Connection.getRepository (/home/loictosser/github.com/n8n/n8n/packages/cli/src/connection/Connection.ts:342:29)
    at Object.getRepository (/home/loictosser/github.com/n8n/n8n/packages/cli/src/index.ts:286:55)
    at Object.init (/home/loictosser/github.com/n8n/n8n/packages/cli/src/Db.ts:178:28)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:31900) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:31900) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:31900) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
(node:31900) UnhandledPromiseRejectionWarning: Error: There was an error: No repository for "CredentialsEntity" was found. Looks like this entity is not registered in current "default" connection?
    at Object.error (/home/loictosser/github.com/n8n/n8n/packages/cli/node_modules/@oclif/errors/lib/index.js:26:15)
    at Start.error (/home/loictosser/github.com/n8n/n8n/packages/cli/node_modules/@oclif/command/lib/command.js:60:23)
    at /home/loictosser/github.com/n8n/n8n/packages/cli/commands/start.ts:206:10
(node:31900) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

**To Reproduce**
1. put postgres as database
2. Run
3. Silently fail

**Expected behavior**
No error for entities.
Maybe a debug mode

**Environment (please complete the following information):**
 - OS: Archlinux
 - n8n Version master
 - Node.js Version 14.5.0

**Comment by janober** [comment]
It looks like the error you are seeing is unrelated and has to do with an "await" being missing in front of `createConnection` in your changed code. Because it does not wait, it does not get registered and that is why it throws this error. We will, however, check out why it does not work in general.

**Comment by janober** [comment]
A quick Google search brought me to this issue:
https://github.com/typeorm/typeorm/issues/5949

So I upgrade the pg package to the latest version and it works now also fine with Node.js v14.

Will get released with the next n8n version.

**Comment by janober** [comment]
Got released with n8n@0.75.0
